### PR TITLE
Semi auto export bin + several UI improvements

### PR DESCRIPTION
**Pull request by matbme**
This PR adds a prompt for the user to automatically export binaries installed via Apx (although the current message is horrible and I could use some ideas), and also replaces all previous instances of spinner that used a dedicated library with Orchid ones ~~because ours is better~~.

### CODE REVIEW

File: core/pkgmanager.go [hunk 1]
```diff
@@ -274,6 +274,41 @@ func (c *Container) IsPackageInstalled(pkgname string) (bool, error) {
 	return result_bool, nil
 }
 
+func (c *Container) BinariesProvidedByPackage(pkgname string) ([]string, error) {
```
Comment (bketelsen) [inline core/pkgmanager.go:277]: Beautiful!

File: core/pkgmanager.go [hunk 2]
```diff
@@ -274,6 +274,41 @@ func (c *Container) IsPackageInstalled(pkgname string) (bool, error) {
...
+	var query_cmd string
+	switch c.containerType {
+	case APT:
+		query_cmd = "dpkg -L %s | grep /usr/bin/ | cut -f 4 -d /"
+	case AUR:
+		query_cmd = "pacman -Ql %s | grep /usr/bin/. | cut -f 4 -d /"
+	case DNF:
+		query_cmd = "rpm -ql %s | grep /usr/bin/ | cut -f 4 -d /"
+	case APK:
+		query_cmd = "apk info -L neovim | grep usr/bin/ | cut -f 3 -d /"
```
Comment (bketelsen) [inline core/pkgmanager.go:287]: missed a %s here

File: core/pkgmanager.go [hunk 3]
```diff
@@ -274,6 +274,41 @@ func (c *Container) IsPackageInstalled(pkgname string) (bool, error) {
...
+	case ZYPPER:
+		query_cmd = "rpm -ql %s | grep /usr/bin/ | cut -f 4 -d /"
+	case XBPS:
+		query_cmd = "xbps-query -f neovim | grep /usr/bin/ | cut -f 4 -d /"
```
Comment (bketelsen) [inline core/pkgmanager.go:291]: and here

**Review by bketelsen — CHANGES_REQUESTED** [review]
delightful change! just a few missing string substitutions to fix, then LGTM

**Review by kbdharun — APPROVED** [review]

**Review by MonsterObserver — APPROVED** [review]
LGTM

**Pull request merged**